src/test/java/hudson/plugins/throttleconcurrents/ThrottleStepTest.java

```diff
@@ -98,6 +98,7 @@ public void onePerNode() throws Exception {
         assertThat(
                 blockageReasons,
                 hasItem(Messages._ThrottleQueueTaskDispatcher_MaxCapacityOnNode(1).toString()));
+        TestUtil.hasPauseActionForItem(queuedItem);
```

Comment:
This assertion needs to be added to other tests as well, at least in `ThrottleStepTest` (if not also the other Pipeline tests).

Reply:
Done